http/server.lua

```diff
@@ -1295,11 +1296,80 @@ local function url_for_httpd(httpd, name, args, query)
     end
 end
 
+local function create_ssl_ctx(host, port, ssl_opts, use_tls)
+    if use_tls ~= true then
+        return nil
+    end
```

Comment:
A strange arg and usage for function with name `create_something`.
```suggestion
local function create_ssl_ctx(host, port, ssl_opts)
```

Reply:
Deleted this argument.